lib/ship.rb

```diff
@@ -0,0 +1,22 @@
+class Ship 
+  attr_reader :name,
+              :length,
+              :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
+    @health = length
+    @sunk = false
+  end
+
+  def sunk? 
+    @sunk
+  end
+
+  def hit 
+    @health -= 1
+    @sunk = true if @health == 0
+  end
+
+end
```

Comment:
What's weird is that it thinks it's adding all this code to the ship.rb and considers the current file "blank"... even though the github origin main has ship.rb populated.

It _is_, however, the exact same as your ship.rb... so we aren't messing anything up